src/Client.php

```diff
@@ -427,7 +427,7 @@ public function update(string $collection, array $where = [], array $updates = [
 
         $this->query(
             array_merge([
-                MongoCommand::UPDATE => $collection,
+                Command::UPDATE => $collection,
```

Comment:
```suggestion
                self::COMMAND_UPDATE => $collection,
```

No need to have the Command class we can just have all constants in this `Client` class.

Reply:
Done